hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -219,13 +230,13 @@ public static JavaSparkContext buildSparkContext(String appName, String sparkMas
   /**
    * Build Hoodie write client.
    *
-   * @param jsc Java Spark Context
-   * @param basePath Base Path
-   * @param schemaStr Schema
+   * @param jsc         Java Spark Context
```

Comment:
are these from intellij formatting...

Reply:
yes, these might cause by the different settings when I switched machines. I removed all `this` and `final` manually instead of reverting commit, so there are still a few lines changing due to the initial formatting. I can revert those comments as well if it's preferable.

Reply:
Thanks for doing it.. looks okay for now.